frontend/src/lib/api/auth.ts

```diff
@@ -0,0 +1,26 @@
+import { AxiosPromise } from 'axios'
+import Cookies from 'js-cookie'
+
+import { CurrentUserResponse } from '../../types/currentUserResponse'
+import { SignInParams } from '../../types/signInParams'
+import { SignInResponse } from '../../types/signInResponse'
+import { SignUpParams } from '../../types/signUpParams'
+import { SignUpResponse } from '../../types/signUpResponse'
+import client from './client'
+
+export const signUp = (params: SignUpParams, token?: string | null): AxiosPromise<SignUpResponse> =>
+  client.post('auth', params, { headers: { InvitationToken: token || '' } })
```

Comment:
`post(...)` がスラッシュありなしのどちらで受け付けてくれるのか考えるの大変なので、一律スラッシュを付けるようにしておいたほうが良いかもしれません。

```suggestion
  client.post('/auth', params, { headers: { InvitationToken: token || '' } })
```

Reply:
確かにそうですね！！スラッシュありで統一しました😊👍